tests/test_updatingrelationship.py

```diff
@@ -23,6 +23,7 @@
 from sqlalchemy.orm import backref
 from sqlalchemy.orm import relationship
 
+from flask.ext.restless import ProcessingException
```

Comment:
Please add a blank line after this import, to separate imports from the package under test from the imports from test helpers.